### DISCUSSION TITLE
post/vite-ts-react-path

### DISCUSSION THREAD

**Discussion by giscus** [COYO-HM/COYO-HM.github.io#130] [Comments]
# post/vite-ts-react-path

Vite 환경에서 TypeScript 또는 JavaScript인 경우 절대 경로 설정하는 방법

https://coyo-hm.github.io/post/vite-ts-react-path

**Comment by PerlPark** [comment]
감사합니다 덕분에 해결했습니다~
그런데 `vite-tsconfig-paths 모듈 미사용시` 코드에 tsconfigPaths가 잘못 들어가 있어요
```
plugins: [react(), tsconfigPaths()],
```

**Reply by coyo-hm** [reply]
알려주셔서 감사합니다. 수정 완료했습니다!!
